docs/quarantine.md

```diff
@@ -87,22 +87,27 @@ test itself doesn't need changes to be fixed (maybe the fix needs to be done on
 other parts of the code base or in a separate repo). In any case, the team
 assigned must communicate when the test is expected to be stable.
 
-A test must be put out of quarantine when:
-* It hasn't failed on any of the periodic lanes in the two weeks after the time
-indicated by the team assigned to bring the test back to the stable suite.
+A test is eligible for de-quarantining once it demonstrates a failure rate of 0.1% or less.
 
+If the SIG believes the flakyness is resolved they can trigger the pull-kubevirt-check-dequarantine-test job by
+commenting /test pull-kubevirt-check-dequarantine-test[^1] directly on the dequarantine pull request.
 
-After two weeks with successful executions has passed, a quarantined tests will
-be ready to join the stable suite again. A member of the team assigned to each
-quarantined tests will propose a PR to remove the text `[QUARANTINE]` and the
+If the periodic test runs confirm the failure rate meets the above criteria the SIG can open a dequarantine PR.
```

Comment:
The `if` can be dropped

Reply:
No, because the entire paragraph has changed, see above